### ISSUE TITLE
TOML Comment disappearing

### ISSUE THREAD

**Issue by t-boom** [TheElectronWill/Night-Config#29]
I found one more place where comments are not being kept, but I'm honestly not sure where they would be kept. The following
```toml
hosts = [
  "alpha", # Where does this comment go?
  "omega"
]
```
loses its comment and becomes
```toml
hosts = ["alpha", "omega"]
```

(Sorry for all the issues, but I really do like your library)

**Comment by TheElectronWill** [comment]
This is because the comments are stored per config value, not for the value's content.  So the comments inside the array are ignored.

They could be kept by changing the comments' storage to a slightly more complicated one. In particular, `getComment(path)` would return a CommentInfos object instead of a String, and this object would contain the comments of the array's elements.

**Comment by t-boom** [comment]
Makes sense. This (along with the related issues) aren't a deal breaker, so it's probably not worth implementing if the comment storage would be too complex.
